graphql-dgs/src/main/kotlin/com/netflix/graphql/dgs/internal/DgsSchemaProvider.kt

```diff
@@ -59,7 +59,8 @@ class DgsSchemaProvider(
     private val applicationContext: ApplicationContext,
     private val federationResolver: Optional<DgsFederationResolver>,
     private val existingTypeDefinitionRegistry: Optional<TypeDefinitionRegistry>,
-    private val mockProviders: Optional<Set<MockProvider>>
+    private val mockProviders: Optional<Set<MockProvider>>,
+    private val basedir: String = "schema"
```

Comment:
@marceloverdijk I am sorry I totally missed this MR and just saw it now. Better late than never, eh? 

Can we remove this default from the constructor so that the default is only in the config props?

Reply:
No problem. I will be working on an alternative PR this week anyway.

Reply:
Cool. I look forward to it.